src/components/Graph/MaxPrice.tsx

```diff
@@ -0,0 +1,64 @@
+import { HistoryData } from '@shapeshiftoss/types'
+import { AxisScale } from '@visx/axis'
+import { Text } from '@visx/text'
+
+export interface LineChartProps {
+  data: HistoryData[]
+  xScale: AxisScale<number>
+  yScale: AxisScale<number>
+  width: number
+  yMax: number
+  label: string
+  xDate: Date
+  yText: number
+  xText: number
+  margin: { top: number; right: number; bottom: number; left: number }
+  hideBottomAxis?: boolean
+  stroke: string
+  hideLeftAxis?: boolean
+  top?: number
+  left?: number
+  children?: React.ReactNode
+  xTickFormat?: (d: any) => any
+}
+
+export const MaxPrice = ({
+  data,
+  label,
+  yText,
+  yScale,
+  xScale,
+  stroke,
+  width,
+  xDate,
+  xText,
+  margin
+}: LineChartProps) => {
+  const handleTextPos = (x: number): { x: number; anchor: 'end' | 'start' | 'middle' } => {
+    const offsetWidth = width / 2
+    const buffer = 16
+    const end = width - offsetWidth
+    if (x < offsetWidth) {
```

Comment:
Since offsetWidth isn't reused, is it more clear to get rid of the intermediate variable and just use `end` instead?

```
    const buffer = 16
    const end = width / 2
    if (x < end) {
    ```